scripts/create-a-m-o/functions.js

```diff
@@ -67,6 +67,7 @@ const createFiles = (store, a, m, o, done) => () => {
   templateJson.name = `@axa-ch/${fileName}`;
   templateJson.version = '0.0.0-beta.0';
   templateJson.homepage = `https://github.com/axa-ch/patterns-library/tree/develop/src/components/${folderMap[type]}/${fileName}#readme`;
+  templateJson.description = `The ${fileName} component for the AXA Pattern Library`;
```

Comment:
You could also remove the default description of `> TODO: description` in `/patterns-library/scripts/create-a-m-o/templates/template-package.json`

Reply:
hä? das hani jo gmacht

Reply:
sorry, hani überseh